src/asset/hollowknight/categories/all-dream-bosses-1432plus.json

```diff
@@ -0,0 +1,69 @@
+{
+    "categoryName": "All Dream Bosses",
+    "splitIds": [
+        "VengefulSpirit",
+        "EnterGreenpath",
+        "MothwingCloak",
+        "BrettaRescued",
+        "EnterSanctum",
+        "DesolateDive",
+        "StoreroomsStation",
+        "CrystalHeart",
+        "BlueLake",
+        "DreamNail",
+        "NightmareLantern",
+        "Flame2",
+        "ShadeSoul",
+        "SoulTyrantEssence",
+        "Flukenest",
+        "DungDefender",
+        "BrokenVessel",
+        "LostKinEssence",
+        "FailedChampionEssence",
+        "Grimmchild2",
+        "TransFlame3",
+        "KingsBrand",
+        "ShadeCloak",
+        "TroupeMasterGrimm",
+        "EnterAnyDream",
+        "WhiteFragmentLeft",
+        "EnterAnyDream",
+        "GalienEssence",
+        "Zote2",
+        "TransFlame3",
+        "EnterAnyDream",
+        "WhiteDefenderEssence",
+        "ColosseumBronzeUnlocked",
+        "ZoteKilled",
+        "GreyPrinceEssence",
+        "WhitePalaceEntry",
+        "Kingsoul",
+        "VoidHeart",
+        "NightmareKingGrimm",
+        "HollowKnightDreamnail"
+    ],
+    "names": {
+        "CrystalHeart": "[2U] %s",
+        "DreamNail": "Drömnålen",
```

Comment:
please remove this override

Reply:
Mb, I used the import splits to generate the json from my file and forgot to delete a couple renames